### ISSUE TITLE
Keystroke not showing with Visual Studio

### ISSUE THREAD

**Issue by neway003** [Code52/carnac#179]
While I'm working with visual studio 2015 Community Edition for creating a screen capture video tutorial, Keystrokes are not showing. But outside visual studio its working.

**Comment by shiftkey** [comment]
@neway003 if you are running Visual Studio in Administrator mode, Carnac is unable to hook into the keystroke events (because it's an elevated process). That's what I suspect is happening.

**Comment by neway003** [comment]
@shiftkey Yes that is the exact issue, Just now I tried it. On administrative mode its not working but normal mode its working. Thank you for your response.
